front/app/components/admin/GraphCards/ReactionsByTimeCard/useReactionsByTime/index.ts

```diff
@@ -13,19 +18,18 @@ import { getFormattedNumbers } from 'components/admin/GraphCards/_utils/parse';
 
 // typings
 import { QueryParameters, Response } from './typings';
-import useAnalytics from 'api/analytics/useAnalytics';
-import useLiveReactionsByTime from './useLiveReactionsByTime';
-import usePublishedReactionsByTime from './usePublishedReactionsByTime';
-import { useMemo, useState } from 'react';
 
 export default function useReactionsByTime({
-  projectId,
+  projectId: _projectId,
   startAtMoment,
   endAtMoment,
   resolution,
 }: QueryParameters) {
   const { formatMessage } = useIntl();
-  const [currentResolution, setCurrentResolution] = useState(resolution);
+  const [currentResolution, _setCurrentResolution] = useState(resolution);
+
+  const { id: graphId } = useNode();
+  const { reportId } = useParams<{ reportId: string }>();
```

Comment:
Will we need to prepare `graphId` and `reportId` in every graph? I was wondering if it's a good reason to move it to the hook.

Reply:
There are more things that don't really make sense now- this 'publised' logic is actually only relevant for the widget and not the graph card, but it lives in the graph card.

Maybe let's wait until we change a few more of these widgets and then decide how/if we want to change this folder structure?

Reply:
Sure.